Enable: true
# 角度范围
AngleRanges:
  [
    10067,10184,
  
    11734,12051,  11801,11968,   11801,11951, 11868,11918, 11784,11951,  11951,11968,
    
    9400,9700,   9517,9567,  9517,9584,
    
    8967,9100,  9000,9067,
    
    6249,6333,  6266,6316,
    
    3015,3115,
    
  ]
# 通道范围
ChannelRanges:
  [
    238,248,
    
    228,254,  194,203,  131,140,  172,174,  157,169,   214,219,
    
    232,251,   184,185,  208,209,
    
    3,4,   34,35,
    
    236,256,   215,216,
    
    33,34,
  ]

Config_1: 1
Config_2: 0.3

DiscardPoint: [3015,3098,1]
